### PR TITLE
SONARJAVA-3892 Exclude "com.sun.jersey" and "com.sun.faces" from S1191 by default

### PR DESCRIPTION
**Pull request by quentin-jaquier-sonarsource**

### CODE REVIEW

File: java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import com.sun.jersey.api.client.ClientHandlerException; // Compliant
 import java.util.ArrayList;
 
-class A {
+class SunPackagesUsedCheck {
```
Comment (dorian-burihabwa-sonarsource) [inline java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java:5]: We should rename the class to `SunPackagesUsedCheckCustom` to match the file name and avoid confusion when looking at the two test files side by side.

File: java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java [hunk 2]
```diff
@@ -2,7 +2,7 @@
 import com.sun.jersey.api.client.ClientHandlerException; // Compliant
 import java.util.ArrayList;
```
Comment (dorian-burihabwa-sonarsource) [inline java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java:3]: We should add a non compliant import from `com.sun.faces.*` or `com.sun.jersey.*" element to show that the `exclude` pattern is overwritten and not extended on top of its default value.

**Review by dorian-burihabwa-sonarsource — CHANGES_REQUESTED** [review]
LGTM but a few tweaks could help the test source files easier to understand

File: java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java (hunk 2 above)
Comment (quentin-jaquier-sonarsource) [reply java-checks-test-sources/src/main/files/non-compiling/checks/SunPackagesUsedCheckCustom.java:3]: Good idea

**Review by dorian-burihabwa-sonarsource — APPROVED** [review]

**Pull request merged**